pkg/reportingplugins/mercury/aggregate_functions.go

```diff
@@ -128,7 +130,7 @@ func GetConsensusLinkFee(paos []PAOLinkFee, f int) (*big.Int, error) {
 	var validLinkFees []*big.Int
 	for _, pao := range paos {
 		fee, valid := pao.GetLinkFee()
-		if valid {
+		if valid && fee.Cmp(Zero) >= 0 {
```

Comment:
You could use `Sign` for these: https://pkg.go.dev/math/big#Int.Sign
```suggestion
		if valid && fee.Sign() >= 0 {
```

Reply:
Cool didn't know about this one